tests/test_regression_test.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pytest
+import shutil
+import subprocess
+import json
+from collections import namedtuple
+import arpeggio
+
+from arpeggio.tests.helpers.utils import process_arpeggio_pair
+
+"""Regression test to check if results provided by original implementation
+of Arpeggio are exactly the same as with refactored version with the use
+of mmcif files.
+"""
+
+AtomTypes = namedtuple('AtomTypes', ['name', 'types'])
+
+
+class Config:
+    """Test configuration. Python 2 environment with dependencies installed
+    is hardcoded. So if required needs to be changed.
+    """
+
+    def __init__(self):
+        root = os.path.join(os.path.dirname(arpeggio.__file__))
+
+        self.structures_path = os.path.join(root, 'tests', 'test_data', 'structures')
+        self.arpeggio = os.path.join(root, 'tests', 'test_data', 'old_arpeggio', 'arpeggio.py')
+        self.py2 = '/Users/lpravda/anaconda3/envs/py2/bin/python'
```

Comment:
Remove this line. If these tests are no longer valid then remove them, but if they are then ensure they all work and we don't have such hardcoded paths because they obviously don't work in CI

Reply:
Removed the older version of arpeggio and tests associated with them.